samples/fleetprovisioning.py

```diff
@@ -263,7 +262,8 @@ def waitForRegisterThingResponse():
             on_connection_interrupted=on_connection_interrupted,
             on_connection_resumed=on_connection_resumed,
             clean_session=False,
-            keep_alive_secs=6)
+            keep_alive_secs=6,
```

Comment:
in the `if args.use_websocket == True:` branch above this, you should also change:
~~websocket_proxy_options=proxy_options,~~ -> http_proxy_options=proxy_options